code/game/objects/items/weapons/storage/storage.dm

```diff
@@ -481,10 +481,10 @@
 	return ..()
 
 /obj/item/storage/emp_act(severity)
-	if(!istype(loc, /mob/living))
-		for(var/obj/O in contents)
-			O.emp_act(severity)
 	..()
+	for(var/i in contents)
+		var/atom/A = i
```

Comment:
I would hope a non-object can't end up in storage...but you know how things go. Not that it matters a whole bunch, but..

Reply:
Shouldn't happen but then again this BYOND